datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -32,4 +32,15 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="add_column_for_user_deactivation" author="ysheiko">
+        <addColumn tableName="user">
+          <column name="active" type="boolean"/>
+        </addColumn>
+        <addDefaultValue tableName="user" columnName="active" defaultValueBoolean="true"/>
```

Comment:
In order to make it more compact, we can write it in the following way:
`<column name="active" type="boolean" defaultValueBoolean="true"/>`
and delete the next row for `addNotNullConstraint`